File: SpringBoot/src/main/java/com/hospital/qlbv/entity/MedicalService.java
package com.hospital.qlbv.entity;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "medical_services")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class MedicalService {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer serviceId;
    
    @Column(name = "service_name", nullable = false)
    private String serviceName;
    
    private String description;
    
    private Double price;
    
    @Column(name = "department_id")
    private Integer departmentId;
    
    private String status;
}

File: Backend/Models/MedicalService.cs
using System.ComponentModel.DataAnnotations;

namespace QuanLyBenhVien.API.Models;

public class MedicalService
{
    [Key]
    public int ServiceID { get; set; }
    public string ServiceName { get; set; } = string.Empty;
    public string? Description { get; set; }
    public decimal? Price { get; set; }
    public int? DepartmentID { get; set; }
    public string? Status { get; set; }
}
